### ISSUE TITLE
Add helper to detect Dynamic Arrays support in Excel

### ISSUE THREAD

**Issue by govert** [Excel-DNA/ExcelDna#292]
Something like this, maybe on `ExcelDnaUtil`:

```cs
        static bool? _supportsDynamicArrays;  
        static bool SupportsDynamicArrays()
        {
            if (!_supportsDynamicArrays.HasValue)
            {
                try
                {
                    var result = XlCall.Excel(614, new object[] { 1 }, new object[] { true });
                    _supportsDynamicArrays = true;
                }
                catch
                {
                    _supportsDynamicArrays = false;
                }
            }
            return _supportsDynamicArrays.Value;
        }
```

**Comment by govert** [comment]
See this suggestion for improvement: https://github.com/Excel-DNA/ExcelDna/wiki/Signing-the-.xll-add-in-binary-for-Excel's-Trust-Center

**Comment by augustoproiete** [comment]
Resolved by [5a53046](https://github.com/Excel-DNA/ExcelDna/commit/5a53046982b71b9e594e5e9af402a98249aa6931)
